src/less/filter-button/base/filter-button.less

```diff
@@ -1,3 +1,7 @@
+div.filter-group {
+    .filter-group;
```

Comment:
Minor, but Is this mixin used elsewhere? Or going to be used elsewhere? If not let's just un-mixin this. All looking good to me otherwise.

Reply:
Yes, I was planning to use it when I do the `filter-menu-button`, to make sure both had the `div.filter-group` rules if they were used independently.

Reply:
But, thinking about it, there might be a better way: as a utility class. Thoughts?

Reply:
I think we'll delve into utility classes once we start exploring layout/grid/spacing and common patterns there.

If filter-menu-buttons can be placed side by side (for some reason I was thinking they are only ever displayed in isolation) then this is fine as is.

Reply:
Cool. I'll leave as-is for the time being. It's easy to switch this around or change it.